schema/unsupported_features.json

```diff
@@ -2,27 +2,27 @@
     "setup_types": [
         {
             "name":"virtual",
-            "unsupported_features": ["lyve_pilot"]
+            "unsupported_features": ["lyve_pilot", "cluster_management", "sw_update"]
```

Comment:
please add ssl_upload for all types as part of unsupported list

Reply:
@kupranay done. please review.